src/Core/src/Layouts/Flex.cs

```diff
@@ -990,27 +990,10 @@ public void init(Item item, float width, float height)
 				ordered_indices = null;
 				if (item.ShouldOrderChildren && item.Count > 0)
 				{
-					var indices = new int[item.Count];
-					// Creating a list of item indices sorted using the children's `order'
-					// attribute values. We are using a simple insertion sort as we need
-					// stability (insertion order must be preserved) and cross-platform
-					// support. We should eventually switch to merge sort (or something
-					// else) if the number of items becomes significant enough.
-					for (int i = 0; i < item.Count; i++)
-					{
-						indices[i] = i;
-						for (int j = i; j > 0; j--)
-						{
-							int prev = indices[j - 1];
-							int curr = indices[j];
-							if (item[prev].Order <= item[curr].Order)
-							{
-								break;
-							}
-							indices[j - 1] = curr;
-							indices[j] = prev;
-						}
-					}
+					// The children need to be ordered using a *stable* sort by Order
+					var indices = item.OrderBy(x => x.Order)
+									.Select((value, index) => index)
+									.ToArray();
```

Comment:
I generally like this over the old code, because it's way more understandable. 👍